### PR TITLE
fix: add openapi spec for custom and external metrics types

### PR DESCRIPTION
**Pull request by arajkumar**
Signed-off-by: Arunprasad Rajkumar <arajkuma@redhat.com>

### CODE REVIEW

File: cmd/adapter/adapter.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ import (
 	basecmd "github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/cmd"
 	"github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/provider"
 
+	customexternalmetrics "github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/apiserver"
```
Comment (dgrisonnet) [inline cmd/adapter/adapter.go:33]: this import should be moved next to the other `custom-metrics-apiserver` imports

Comment (arajkumar) [reply cmd/adapter/adapter.go]: Thanks @dgrisonnet. Fixed it.

File: cmd/adapter/adapter.go [hunk 2]
```diff
@@ -41,7 +42,6 @@ import (
 	"k8s.io/client-go/transport"
 	"k8s.io/component-base/logs"
 	"k8s.io/klog/v2"
-
```
Comment (dgrisonnet) [inline cmd/adapter/adapter.go:44]: let's keep this line break, it makes the imports more readable

Comment (arajkumar) [reply cmd/adapter/adapter.go:44]: Ack.

**Pull request merged**